src/lib/venueTimezone.ts

```
@@ -0,0 +1,116 @@
+];
+
+const EARTH_RADIUS_KM = 6371;
+const MATCH_RADIUS_KM = 50;
+
+function toRad(deg: number): number {
+  return (deg * Math.PI) / 180;
+}
+
+function haversineKm(
+  lat1: number, lng1: number, lat2: number, lng2: number,
+): number {
```

ESLint

[error] Replace `⏎··lat1:·number,·lng1:·number,·lat2:·number,·lng2:·number,⏎` with `lat1:·number,·lng1:·number,·lat2:·number,·lng2:·number`

(prettier/prettier)